extensions/amp-story/1.0/amp-story.js

```diff
@@ -1699,6 +1699,13 @@ export class AmpStory extends AMP.BaseElement {
     const vState = this.getAmpDoc().getVisibilityState();
     const isPreview = vState === VisibilityState_Enum.PREVIEW;
     const isVisible = vState === VisibilityState_Enum.VISIBLE;
+
+    if (isPreview) {
```

Comment:
Will this run on page load, or only when the state changes?

Reply:
I've updated `amp-story-system-layer.js` to hide the system layer via the `i-amphtml-story-hidden` class, but it's not working as expected. The issue is that loading into `preview` mode correctly results in a hidden system layer, but transitioning to `visible` mode does not result in the system layer becoming shown.

Even though I can see that the preview -> visible transition results in `this.storeService_.dispatch(Action.TOGGLE_SYSTEM_UI_IS_VISIBLE, true)` being correctly called within amp-story, `onSystemUiIsVisibleStateUpdate_` fails to be called within amp-story-system-layer

Reply:
Fixed this with my latest commit